platforms/software/testing-base/src/main/java/org/gradle/api/internal/tasks/testing/report/generic/PerRootTabRenderer.java

```diff
@@ -199,33 +203,44 @@ private static void renderSummaryGroup(PerRootInfo info, SimpleHtmlWriter htmlWr
             htmlWriter.endElement();
         }
 
-        private static String getFormattedDuration(SerializableTestResult testResult) {
-            return TimeFormatting.formatDurationVeryTerse(testResult.getDuration());
+        private static String getFormattedDuration(PerRootInfo info) {
+            long minStartTime = Long.MAX_VALUE;
+            long maxEndTime = Long.MIN_VALUE;
+            for (SerializableTestResult result : info.getResults()) {
+                minStartTime = Math.min(minStartTime, result.getStartTime());
+                maxEndTime = Math.max(maxEndTime, result.getEndTime());
+            }
+            return TimeFormatting.formatDurationVeryTerse(maxEndTime - minStartTime);
```

Comment:
❗ This doesn't seem right.  If there are multiple results, and one started at t=1 and took 5 seconds (running until t=6), and another started at t=2 and ran 8 seconds (until t=10), this would result in a displayed duration of 9 seconds...which isn't anything.